### PR TITLE
docs(docs): Update header and footer navigation

### PR DESCRIPTION
**Pull request by Kadee80**
## overview

This PR ensures online documentation navigation header and footer match opentrons.com

## changelog

- remove dead links from resources tab in nav
- update layout template css, js, and navigation

## review requests
1- Checkout this branch
2- `cd api/docs`
3- `make clean html`
4- go to build folder and open any html file 
- [ ] `getting started` and `faq` are removed from resources tab
- [ ] css looks like opentrons.com

go to blog.opentrons.com
- [ ] navigation + css matches opentrons.com

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Opentrons Platform
 
-[![Travis CI][travis-badge]][travis]
+<!-- [![Travis CI][travis-badge]][travis]
```
Comment (mcous) [inline README.md]: Why were these commented out? Looks like the links were fixed

File: README.md [hunk 2]
```diff
@@ -30,16 +30,16 @@ pipette.aspirate(tube_1).dispense(tube_2)
 That is how you tell the Opentrons robot to pipette its max volume from one tube to another. Learn more here:
 
 *   [Documentation](http://docs.opentrons.com)
-*   [Source code](https://github.com/Opentrons/opentrons/tree/v3a/api)
+*   [Source code](https://github.com/Opentrons/opentrons/tree/edge/api)
```
Comment (mcous) [inline README.md]: Use a relative link for this so it works no matter what branch we're on or what the org/repo is named: `[Source code](./api)`

File: README.md [hunk 3]
```diff
@@ -30,16 +30,16 @@ pipette.aspirate(tube_1).dispense(tube_2)
...
 
 ## Opentrons App
 
 Easily upload a protocol, calibrate positions, and run your experiment from your computer.
 
 *   [Documentation](https://support.opentrons.com/)
-*   [Source code](https://github.com/Opentrons/opentrons/tree/v3a/app)
+*   [Source code](https://github.com/Opentrons/opentrons/tree/edge/app)
```
Comment (mcous) [inline README.md]: Same comment about relative links

File: api/docs/source/templates/layout.html [hunk 4]
```diff
@@ -71,9 +71,11 @@
          Products
         </a>
         <ul>
-          <li><a href="https://opentrons.com/robots" >Robots</a></li>
+          <li><a href="https://opentrons.com/robots" >OT-One</a></li>
+          <li><a href="https://opentrons.com/ot-2" >OT-2</a></li>
+          <li><a href="https://opentrons.com/pipettes" >OT-2 Pipettes</a></li>
           <li><a href="https://opentrons.com/modules" >Modules</a></li>
-          <li><a href="https://opentrons.com/ot-app" >OT App</a></li>
+          <li><a href="https://opentrons.com/ot-app" >OT Run App</a></li>
```
Comment (mcous) [inline api/docs/source/templates/layout.html]: Official branding is "Opentrons App"

File: api/docs/source/templates/layout.html [hunk 5]
```diff
@@ -173,9 +174,11 @@ <h3>Sign Up For Our Newsletter</h3>
     <section class="footer-menu">
       <ul>
         <li>Products</li>
-        <li><a href="https://opentrons.com/robots" >Robots</a></li>
+        <li><a href="https://opentrons.com/robots" >OT-One</a></li>
+        <li><a href="https://opentrons.com/ot-2" >OT-2</a></li>
+        <li><a href="https://opentrons.com/pipettes" >OT-2 Pipettes</a></li>
         <li><a href="https://opentrons.com/modules" >Modules</a></li>
-        <li><a href="https://opentrons.com/ot-app" >OT App</a></li>
+        <li><a href="https://opentrons.com/ot-app" >OT Run App</a></li>
```
Comment (mcous) [inline api/docs/source/templates/layout.html]: Opentrons App

**Comment by mcous** [comment]
Also, in terms of commit messages, we don't need scope for this one. When you squash, you can simplify to `docs: ...`

File: api/docs/source/templates/layout.html (hunk 5 above)
Comment (Kadee80) [reply api/docs/source/templates/layout.html]: This one I'm going to float past pantea. I'm just matching main site naming.

File: README.md (hunk 1 above)
Comment (Kadee80) [reply README.md]: Oversight on my part. Will uncomment in next commit

File: api/docs/source/templates/layout.html (hunk 5 above)
Comment (Kadee80) [reply api/docs/source/templates/layout.html]: Got confirmation from Pantea. Changing to OT App

**Pull request merged**